website/docs/content/punctuation/punctuation.md

```diff
@@ -162,8 +342,37 @@ title: Punctuation and special symbols
 * in tooltips that serve as labels for icons or display the element’s full name
 * after URLs
 
-![](/content/punctuation/static/noperiod.png)
-
+<table class="dosdonts">
+    <tr>
+        <th>Don’t</th>
+        <th>Do</th>
+    </tr>
+    <tr>
+        <td>
+            ☑️ Remind later.
+        </td>
+        <td>
+            ☑️ Remind later
+        </td>
+    </tr>
+    <tr>
+        <td>
+            ???
```

Comment:
I'll need help adding an example in a form of an image. But it's small, you can barely see anything, so maybe you'll have a better idea

Reply:
I'll look at it in a moment